HopsanGUI/PlotTab.cpp

```diff
@@ -1397,6 +1397,11 @@ void PlotTab::openFrequencyAnalysisDialog(PlotCurve *pCurve)
     double minX = pCurve->getSharedTimeOrFrequencyVariable()->minOfData();
     double maxX = pCurve->getSharedTimeOrFrequencyVariable()->maxOfData();
 
+    if(minX == maxX) {
```

Comment:
Should not do == commparisson for doubles

Reply:
I'm not sure if it matters when the purpose of the check is to prevent division by zero, but I created an `xdiff` variable instead and compared it to zero.

Reply:
yes seems OK in this case, I did not consider the purpose of the check